tests/utils/testrunner.go

```diff
@@ -0,0 +1,73 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package utils
+
+import (
+	kube "github.com/dapr/dapr/tests/platforms/kubernetes"
+)
+
+// runnerFailExitCode is the exit code when test runner setup is failed
+const runnerFailExitCode = 1
+
+// testingMInterface interface is used for testing TestRunner
+type testingMInterface interface {
+	Run() int
+}
+
+// testingPlatform defines the testing platform for test runner
+type testingPlatform interface {
```

Comment:
I think we can remove 'testing' prefix as this is under test utils package and same applies for testingInterface,

Reply:
I will rename it.

Reply:
I renamed from testinPlatform to PlatformInterface because Platform is used in TestRunner as a member.